src/usernotifications.cs

```diff
@@ -163,6 +167,43 @@ public enum UNShowPreviewsSetting : long
 		Never
 	}
 
+	[iOS (15,0), Mac (12,0), MacCatalyst (15,0), TV (15,0), Watch (8,0)]
+	[Native]
+	public enum UNNotificationInterruptionLevel : long
+	{
+		Active,
+		Critical,
+		Passive,
+		TimeSensitive,
+	}
+
+
+	[MacCatalyst (15,0), NoiOS, NoMac, NoWatch, NoTV]
+	enum UNNotificationInterruptionReason {
+		[Field ("UNNotificationInterruptionReasonAccountSecurity")]
+		AccountSecurity,
+		[Field ("UNNotificationInterruptionReasonHealthAndWellness")]
+		HealthAndWellness,
+		[Field ("UNNotificationInterruptionReasonLocalSafety")]
+		LocalSafety,
+		[Field ("UNNotificationInterruptionReasonHomeSecurity")]
+		HomeSecurity,
+		[Field ("UNNotificationInterruptionReasonOrderPickup")]
+		OrderPickup,
+		[Field ("UNNotificationInterruptionReasonPackageDelivery")]
+		PackageDelivery,
+		[Field ("UNNotificationInterruptionReasonTransportationDeparting")]
+		TransportationDeparting,
+		[Field ("UNNotificationInterruptionReasonSchedule")]
+		Schedule,
+		[Field ("UNNotificationInterruptionReasonTimeExpired")]
+		TimeExpired,
+		[Field ("UNNotificationInterruptionReasonApprovalRequest")]
+		ApprovalRequest,
+		[Field ("UNNotificationInterruptionReasonUrgentMessage")]
+		UrgentMessage,
```

Comment:
Apple, the gift that keeps on giving, only Catalyst??

Reply:
Yup lol it seems so

Reply:
Lol B3 removed this!

Reply:
I suspected something of the kind ;)